neptune_mlflow/data_loader.py

```diff
@@ -69,7 +71,7 @@ def _create_neptune_experiment(self, experiment, run):
             handle_uncaught_exceptions=True
         ) as neptune_exp:
             if run.info.artifact_uri.startswith('file:/'):
-                artifacts_path = run.info.artifact_uri[6:]
+                artifacts_path = url2pathname(urlparse(run.info.artifact_uri).path)
```

Comment:
```
>>> url2pathname(urlparse("file://a/b/c").path)
'/b/c'
```
Chyba nie tak powinno być.

Reply:
Z tego co znalazłem w dokumentacji w trakcie klepania to te artifact_uri to jest absolute path (tak jak w naszych artefaktach). Dla absolute to jak najbardziej działa:
```>>> url2pathname(urlparse("file:///a/b/c").path)
'/a/b/c'
```

Reply:
Dosłownie linijkę wyżej masz ifa jakiego rodzaju są to ścieżki:
```
if run.info.artifact_uri.startswith('file:/'):
```

Reply:
Jeśli mówisz o dokumentacji MLFlow to może ten stary kod w ogóle nie ma sensu, ale ta zmiana też.

Reply:
> Dosłownie linijkę wyżej masz ifa jakiego rodzaju są to ścieżki:

Ten if sprawdza tylko czy to jest `file` vs `s3` (lub jakis inny remote bo tego nie obsługujemy) i raczej powinno być sprawdzane przez schemę.

>  Jeśli mówisz o dokumentacji MLFlow to może ten stary kod w ogóle aktualnie nie ma sensu, ale ta zmiana też.

A to trochę też prawda :D

Reply:
To ja w końcu nie rozumiem na czym stoimy. W moim komentarzu stwierdzam tylko, że:
```
>>> url2pathname(urlparse("file://a/b/c").path)
'/b/c'
```
to daje inny wynik niż dawało w starym kodzie, w którym usuwaliśmy tylko pierwsze 6 znaków czyli `file:/`, a teraz w wyniku brakuje również `/a`)

Reply:
Dobra. To w skrócie rozumiem, że wolisz abym to przepisał tak aby działały też ścieżki relatywne bo potencjalnie mogły być wcześniej supportowane?

Reply:
`file://a/b/c` - to jest relatywna ścieżka?
Btw. czemu w ogóle się zajmujemy tym MLFlowem. Nie wiem czy to nie jest jakaś kompletnie deprecated integracja.

Reply:
WTF. Sformatowało mi jakoś źle komentarz i teraz widzę, że mogłem Ciebie nieźle skonfundować. Ta którą podajesz to jest relatywna ścieżka. Nierelatywna ma 3 slashe lub nazwe serwera (a tego nie obsługujemy). https://stackoverflow.com/a/15193744
Mój przykład był dla `url2pathname(urlparse("file:///a/b/c").path)`